Codegen/src/Nodes/TypedDataNode.cs

```diff
@@ -972,11 +972,15 @@ private static System.Text.StringBuilder GetIndent(int level, int indent, bool i
 
         };
 
-        public static List<ITextLine> InsertChildren(ColumnsLayout? layout, List<string> rootProcedures, List< Tuple<string,string> > rootVariableName, DataDefinition ownerDefinition, DataDefinition type, int level, int indent)
+        public static List<ITextLine> InsertChildren(ColumnsLayout? layout, List<string> rootProcedures, List<Tuple<string, string>> rootVariableName, DataDefinition ownerDefinition, DataDefinition type, int level, int indent)
```

Comment:
Considering all the usages of the public method, you can merge `DataDefinition ownerDefinition` and `DataDefinition type` as a single parameter and also remove `int indent` parameter.

Reply:
It's right; done.
https://github.com/TypeCobolTeam/TypeCobol/pull/1646/commits/5ec741f218377464909aae42e8454ea4de4d1d2e